pushy/src/main/java/com/eatthepath/pushy/apns/util/ApnsPayloadBuilder.java

```diff
@@ -715,6 +729,76 @@ public ApnsPayloadBuilder addCustomProperty(final String key, final Object value
         return this;
     }
 
+    /**
+     * <p>Sets the provided content state object inside content-state payload for Live Activities.</p>
+     *
+     * <p>The precise strategy for serializing the values of custom properties is defined by the specific
+     * {@code ApnsPayloadBuilder} implementation.</p>
+     *
+     * @param contentState content-state object
+     *
+     * @return a reference to this payload builder
+     *
+     * @see <a href="https://developer.apple.com/documentation/activitykit/update-and-end-your-live-activity-with-remote-push-notifications">
+     *      Updating and ending your Live Activity with remote push notifications</a>
+     */
+    public ApnsPayloadBuilder setContentState(final Map<String, Object> contentState) {
+        this.contentState = contentState;
+        return this;
+    }
+
+    /**
+     * <p>Sets whether the notification payload will be used for updating the Live Activity
+     * or for ending it.</p>
+     *
+     * @param event the type of update event to send to the destination activity
+     *
+     * @return a reference to this payload builder
+     *
+     * @see <a href="https://developer.apple.com/documentation/activitykit/update-and-end-your-live-activity-with-remote-push-notifications">
+     *      Updating and ending your Live Activity with remote push notifications</a>
+     */
+    public ApnsPayloadBuilder setEvent(final LiveActivityEvent event) {
+        this.event = event;
+        return this;
+    }
+
+    /**
+     * <p>Sets a timestamp for the push notification payload.   </p>
+     *
+     * @param timestamp Timestamp
+     *
+     * @return a reference to this payload builder
+     *
+     * @see <a href="https://developer.apple.com/documentation/activitykit/update-and-end-your-live-activity-with-remote-push-notifications">
+     *      Updating and ending your Live Activity with remote push notifications</a>
+     */
+    public ApnsPayloadBuilder setTimestamp(final Instant timestamp) {
+        this.timestamp = timestamp;
+        return this;
+    }
+
+    /**
+     * <p>Sets a timestamp for the push notification payload. The timestamp is used to discard older
+     * push notifications. According to Apple's documentation:</p>
+     *
+     * <blockquote>When you end a Live Activity, by default the Live Activity appears on the Lock Screen for up to
+     * four hours after it ends to allow the user to glance at their phone to see the latest information. To dismiss
+     * the Live Activity from the Lock Screen immediately after it ends, provide a date for "dismissal-date" that’s
+     * in the past. Alternatively, provide a date within a four-hour window to set a custom dismissal date.</blockquote>
+     *
+     * @param dismissalDate Date when the Live Activity will be dismissed
```

Comment:
Minor: teeeeeechnically this isn't a "date" by `java.time` conventions (i.e. a "date" would be something like "November 27, 2022").

```suggestion
     * @param dismissalDate the instant when the Live Activity will be dismissed
```